js/researches/english/passivevoice-english/wordIndicesRegexes.js

```diff
@@ -0,0 +1,51 @@
+var stringToRegex = require( "../../../stringProcessing/stringToRegex.js" );
+var determinerList = require( "./determiners" )();
+var arrayToRegex = require( "../../../stringProcessing/createRegexFromArray.js" );
+
+
+/**
+ * Matches determiners in sentence parts and returns them and their indices.
+ *
+ * @param {string} sentencePart The sentence part to match the determiners in.
+ * @returns {Array} The list of result objects.
+ */
+var determiners = function( sentencePart ) {
+	var results = [];
+	var determinersRegex = arrayToRegex( determinerList );
```

Comment:
I would do this outside this function. In this case the regex will be built every time the function is called.